bugbug/bug_features.py

```diff
@@ -207,6 +207,27 @@ def __call__(self, bug, **kwargs):
         return None
 
 
+class delta_nightly_request_merge(single_bug_feature):
+    def __call__(self, bug, **kwargs):
+        for history in bug["history"]:
+            for change in history["changes"]:
+                if change["added"].startswith("approval-mozilla"):
+                    uplift_request_datetime = datetime.strptime(
+                        history["when"], "%Y-%m-%dT%H:%M:%SZ"
+                    ).replace(tzinfo=timezone.utc)
+
+                    landing_comments = bugzilla.get_landing_comments(
+                        bug["comments"], ["nightly"]
+                    )
+                    nightly_patch = datetime.strptime(
+                        landing_comments[0]["comment"]["creation_time"],
```

Comment:
There might be multiple landings in the same bug, we should measure the time distance between the last landing before the uplift request and the uplift request